superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-return-assign */
```

Comment:
@john-bodley did a lot of work to remove the lint rules that apply to the whole file, since they lead to people breaking good practices unintentionally. Unless there's a good reason, let's move this to the place where it's needed.